os/drivers/serial/serial.c

```diff
@@ -125,6 +139,86 @@ static const struct file_operations g_serialops = {
  * Private Functions
  ************************************************************************************/
 
+#ifdef CONFIG_DYNAMIC_LOGS_STORING
+void read_log_net(void)
+{	size_t free_size;
+
+	if (!sq_empty(&log_net_chunks)) { /* check if there are any logs saved */
+		struct mm_heap_s *myheap = mm_get_heap(sq_peek(&log_net_chunks)); /* get heap details using the head of the chunks linked list */
+		if (myheap == NULL) {
+			lldbg("mm_get_heap failed\n");
```

Comment:
Let us use lldbg only if absolutely necessary. Please check at all places if it is required.

Reply:
Also, do we need to print this always? or we could use llvdbg to print only if verbose logs are enabled?